app/script/conversation/ConversationRepository.js

```diff
@@ -2589,7 +2616,8 @@ z.conversation.ConversationRepository = class ConversationRepository {
     return Promise.resolve()
       .then(() => {
         if (!messageEntity.user().is_me && !messageEntity.ephemeral_expires()) {
-          throw new z.conversation.ConversationError(z.conversation.ConversationError.TYPE.WRONG_USER);
+          const error = new z.conversation.ConversationError(z.conversation.ConversationError.TYPE.WRONG_USER);
+          throw error;
```

Comment:
No need to pull this into a const. This easily fit a single line before. I just thing single line statements breaking into three lines are easier to read if you pull part out into a const.